src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/LogsTests.cs

```diff
@@ -161,6 +173,48 @@ public Task LogsDefaultLevelFallbackTest(DiagnosticPortConnectionMode mode, LogF
                 logFormat);
         }
 
+        /// <summary>
+        /// Test that log events with a category that doesn't have a specified level are collected
+        /// at the log level specified in the request body.
+        /// </summary>
+        [ConditionalTheory(nameof(SkipOnWindowsNetCore31))]
+        [InlineData(DiagnosticPortConnectionMode.Connect, LogFormat.JsonSequence)]
+        [InlineData(DiagnosticPortConnectionMode.Connect, LogFormat.NDJson)]
+#if NET5_0_OR_GREATER
+        [InlineData(DiagnosticPortConnectionMode.Listen, LogFormat.JsonSequence)]
+        [InlineData(DiagnosticPortConnectionMode.Listen, LogFormat.NDJson)]
+#endif
+        public Task LogsDefaultLevelFallbackActionTest(DiagnosticPortConnectionMode mode, LogFormat logFormat)
```

Comment:
These type of tests that are loading only the CollectLogs action (and depencies) but not running dotnet-monitor should be in the Microsoft.Diagnostics.Monitoring.Tool.UnitTests project.